app/test_utils.go

```diff
@@ -44,8 +44,9 @@ func newTestSetup() testSetup {
 	ms.MountStoreWithDB(keySupply, sdk.StoreTypeIAVL, db)
 	ms.MountStoreWithDB(keyParams, sdk.StoreTypeIAVL, db)
 	ms.MountStoreWithDB(tkeyParams, sdk.StoreTypeIAVL, db)
-	// nolint:errcheck
-	ms.LoadLatestVersion()
+
+	if err := ms.LoadLatestVersion(); err != nil {
```

Comment:
If error ever is nil here you're going to have a panic, should do something inside here

Reply:
Yeah I was debating how to handle it. I ended up replicating the same error handling in `app/ethermint.go`, that is, `cmn.Exit(err.Error())`

Let me know if that is sufficient.

Reply:
Yeah, that works! Just trying to avoid the cryptic error when it's left blank inside the conditional